test_api.py

```diff
@@ -1264,6 +1278,30 @@ def test_delete_with_invalid_host_id(self):
 
         self.delete(url, 400)
 
+    def test_delete_when_host_is_deleted(self):
+        url = HOST_URL + "/" + self.added_hosts[0].id
+        with patch("api.host._get_host_list_by_id_list", self.RaceCondition):
+            # deletion should give back 200 status
+            self.delete(url, 200, return_response_as_json=False)
+
+    def test_delete_when_all_hosts_are_deleted(self):
+        url = HOST_URL + "/" + self.added_hosts[0].id + "/" + self.added_hosts[1].id
```

Comment:
There should be a comma (,) instead of a slash (/).

```suggestion
        url = HOST_URL + "/" + self.added_hosts[0].id + "," + self.added_hosts[1].id
```